src/docs_website/src/docs.zig

```diff
@@ -367,7 +384,7 @@ const Page = struct {
     ) !void {
         const pandoc_step = std.Build.Step.Run.create(b, "run pandoc");
         pandoc_step.addFileArg(website.pandoc_bin);
-        pandoc_step.addArgs(&.{ "--from", "gfm", "--to", "html5" });
+        pandoc_step.addArgs(&.{ "--from", "gfm+smart", "--to", "html5" });
```

Comment:
Please double-check that `+smart` doesn't cause any surprise rendering issues for us, by diffing HTML before and after.

Reply:
OK, I manually went through all of our output files and I think we're good. It's fancier quotes, ellipsis, and dashes. No surprises!

Here's a sample:
<img width="752" alt="image" src="https://github.com/user-attachments/assets/19c8c574-0b49-45e7-9500-b4cf09f1d857" />